web/frontend/src/types/getObjectType.ts

```diff
@@ -1,44 +1,58 @@
 import ShortUniqueId from 'short-unique-id';
-import { Rank, Select, Subject, Text } from './configuration';
+import {
```

Comment:
Something I'm not 100% convinced with, though I'm not that far:
Use instead `import * as types from './configuration';`. It makes the import list smaller, and the code easier to read since you know where the element comes from. For example:
```ts
const newSubject = (): types.Subject => {
```

what do you think ? It also applies on some other files.

Reply:
Yes, good idea, some function might become a bit clunky with all the types.Text etc but I think overall it would make things clearer

Reply:
I would do it only when the import list is big, or when it adds clarity.